source/declarations.tex

```diff
@@ -2713,12 +2702,16 @@
 \terminal{\&} \opt{attribute-specifier-seq} \terminal{D1}\br
 \terminal{\&\&} \opt{attribute-specifier-seq} \terminal{D1}
 \end{ncsimplebnf}
-and the type of the identifier in the declaration
+and the type of the contained
+\grammarterm{declarator-id}
+in the declaration
 \tcode{T}
 \tcode{D1}
 is ``\placeholder{derived-declarator-type-list}
 \tcode{T}'',
-then the type of the identifier of
+then the type of the
+\grammarterm{declarator-id}
+in
 \tcode{D}
 is ``\placeholder{derived-declarator-type-list} reference to
 \tcode{T}''.
```

Comment:
> A *declarator* of the form `&` *attribute-specifier-seq*[opt] *D* or `&&` *attribute-specifier-seq*[opt] *D* determines the type of its declared entity to be the type of the entity declared by the declaration *U* *D*, where *U* is a *type-specifier* that denotes the type ''reference to `T`''.

(And so on for later subclauses.)

Reply:
@zygoloid The removal of _derived-declarator-type-list_ is definitely a good idea, however, moving the form of the declarator to be inline does degrade readability, especially for longer declarator forms (i.e. functions). Perhaps we keep the form of the declarators indented as is done currently, and apply the other changes to the paragraphs?

Reply:
Also, maintaining `T` throughout subclauses is a little awkward, and could lead to confusion (typically, when "variables" are established, we carry them no further than then end of a paragraph, let alone a subclause)